<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Front Motor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="frontBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="frontChBx">
           <property name="text">
            <string>CW/CCW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rear Motor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rearBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rearChBx">
           <property name="text">
            <string>CW/CCW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Left Motor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="leftBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="leftChBx">
           <property name="text">
            <string>CW/CCW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Right Motor:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rightBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rightChBx">
           <property name="text">
            <string>CW/CCW</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="sendValsBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Send Values</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kickBtn">
         <property name="text">
          <string>Kick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="text">
          <string>Stop</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statLabel">
       <property name="text">
        <string>Disconnected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="disconnectBtn">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>frontBox</tabstop>
  <tabstop>rearBox</tabstop>
  <tabstop>leftBox</tabstop>
  <tabstop>rightBox</tabstop>
  <tabstop>frontChBx</tabstop>
  <tabstop>rightChBx</tabstop>
  <tabstop>leftChBx</tabstop>
  <tabstop>disconnectBtn</tabstop>
  <tabstop>connectBtn</tabstop>
  <tabstop>exitBtn</tabstop>
  <tabstop>rearChBx</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
